.golangci.yml

```diff
@@ -1,4 +1,103 @@
 # golangci-lint configuration from https://golangci-lint.run/usage/configuration/
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  concurrency: 4
+
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 2m
+
+  skip-dirs:
+    - .git
+    - .github
+    - bin
+    - charts
+    - docs
+    - hack
+  skip-files:
+    - cmd/allinone/app/entrypoint.go
```

Comment:
Exclude those lint failed files. If anyone confirm that any files are lint-well, you can remove corresponding lines.